prototypeGui.py

```diff
@@ -0,0 +1,393 @@
+        self.setCentralWidget(self.table_widget)
+
+        self.center()
+
+        # Easily close the GUI (without saving)
+        close_gui = QAction("&Close GUI", self)
+        close_gui.setShortcut('Ctrl+Q')
+        close_gui.setStatusTip('Close without saving.')
+        close_gui.triggered.connect(self.closeEvent)
+
+        self.show()
+
+
+    def center(self):
+        """
+        Centers the window on the screen.
+        """
+
+        # Get window geometry.
+        qr = self.frameGeometry()
+        # Get screen resolution.
+        cp = QDesktopWidget().availableGeometry().center()
+        # Move center of rectangle of equal size to geometry
+        #  to the center of the screen.
+        qr.moveCenter(cp)
+        # Move the top left corner of the geometry to the
+        #  top left corner of the rectangle.
+        self.move(qr.topLeft())
+
+
+    def closeEvent(self, event):
+        """
+        Prompts the user before closing window.
+        """
+        reply = QMessageBox.question(self, 'Message',
+            "Are you sure to quit?", QMessageBox.Yes |
+            QMessageBox.No, QMessageBox.Yes)
+
+        if reply == QMessageBox.Yes:
+            event.accept()
+        else:
+            event.ignore()
+
+
+class Table(QWidget):
+    """
+    This class handles creating tabs, their contents, and exporting to YAML.
+    In each tab is file information.
+    When the Export button is clicked, the information
+    entered is exported to a YAML file.
+    """
+
+    def __init__(self, parent):
+        """
+        Sets up tabs and contents.
+        """
+        super(QWidget, self).__init__(parent)
+        self.layout = QVBoxLayout(self)
+
+        # Initialize tab screen
+        self.tabs = QTabWidget()
+        self.tab1 = QWidget()
+        self.tab2 = QWidget()
+        self.tab3 = QWidget()
+        self.tabs.resize(300, 200)
+
+        # Tab 1 contents
+        author_name = QLineEdit()
+        author_orcid = QLineEdit()
+        file_version = QLineEdit('0')
+        chemked_version = QLineEdit('0.0.1')
+
+        doi = QLineEdit()
+        ref_authors = QLineEdit()
+        journal = QLineEdit()
+        year = QLineEdit()
+        volume = QLineEdit()
+        pages = QLineEdit()
+        detail = QLineEdit()
+
+        self.tab1.metadata_values = [author_name, author_orcid, file_version,
+                            chemked_version]
+        self.tab1.reference_values = [doi, ref_authors, journal,
+                            year, volume, pages, detail]
+        self.tab1.metadata_labels = ['Author Name', 'Author ORCID', 'File Version', 'ChemKED Version']
+        self.tab1.reference_labels = ['DOI', 'Author(s)', 'Journal',
+                            'Year', 'Volume', 'Pages', 'Detail']
+
+        self.tab1.vbox = QVBoxLayout()
+        self.tab1.formGroupBox = QGroupBox()
+        self.tab1.formLayout = QFormLayout()
+        for i in range(len(self.tab1.metadata_labels)):
```

Comment:
Rather than creating an index and looping, you can directly loop over the elements that you want with the `zip` function:

```python
for l, v in zip(self.tab1.metadata_labels, self.tab1.metadata_values):
    self.tab1.formLayout.addRow(Qlabel(l), v)
```

and the same for the other loops